src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -646,45 +644,6 @@ private String fillUpInstructorJoinFragment(InstructorAttributes instructor, Str
     }
 
     /**
-     * Generates the system error report email for the given {@code error}.
-     */
-    public EmailWrapper generateSystemErrorEmail(
-            String requestMethod, String requestUserAgent, String requestPath, String requestUrl,
-            String requestParams, UserType userType, Throwable error) {
-
-        String errorMessage = error.getMessage();
-        String stackTrace = TeammatesException.toStringWithStackTrace(error);
-
-        // If the error doesn't contain a short description, retrieve the first line of stack trace.
-        // truncate stack trace at first "at" string
-        if (errorMessage == null) {
-            int msgTruncateIndex = stackTrace.indexOf("at");
-            if (msgTruncateIndex > 0) {
-                errorMessage = stackTrace.substring(0, msgTruncateIndex).trim();
-            } else {
-                errorMessage = "";
-            }
-        }
-
-        String actualUser = userType == null || userType.id == null ? "Not logged in" : userType.id;
-
-        String emailBody = Templates.populateTemplate(EmailTemplates.SYSTEM_ERROR,
```

Comment:
So is `EmailTemplates.SYSTEM_ERROR`, and by extension `src/main/resources/systemErrorEmailTemplate.html`.